mama/jni/src/main/java/com/wombat/mama/examples/MamaSubscriberJava.java

```diff
@@ -369,10 +369,10 @@ public void onMsg(MamaBasicSubscription subscription, MamaMsg msg)
         {
             if( myQuietLevel < 2 )
             {
-                System.out.println ("Reveived msg: Topic=" +
-                                   msg.getString ("PublisherTopic",10002) +
-                                   ", Sequence Nunber="+msg.getU32 ("MdSeqNum", 10001));
-                System.out.println("Received msg:");
+                System.out.println ("Received msg: Topic=" +
+                                   msg.getString ("MdFeedHost",12) +
```

Comment:
That's not the Topic - output will read weird.  You might be better getting the topic from the subscription object and don't bother trying to access the message for it anyway. I assume the problem here was that the source didn't contain that field so this was throwing an exception, so that should solve the problem (if that's the case) without providing misleading output.